scripts/core/exp-command-buffer.yml

```diff
@@ -0,0 +1,309 @@
+returns:
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_EXP
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Decrement the command-buffer object's reference count and delete the command-buffer object if the reference count becomes zero."
+class: $xCommandBuffer
+name: ReleaseExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object"
+returns:
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_EXP
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Stop recording on a command-buffer object such that no more commands can be appended and make it ready to enqueue."
+class: $xCommandBuffer
+name: FinalizeExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object"
+returns:
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_EXP
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Append a kernel execution command to a command-buffer object"
+class: $xCommandBuffer
+name: AppendKernelLaunchExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object"
+    - type: $x_kernel_handle_t
+      name: hKernel
+      desc: "[in] kernel to append"
+    - type: uint32_t
+      name: workDim
+      desc: "[in] dimension of the kernel execution"
+    - type: "const size_t*"
+      name: pGlobalWorkOffset
+      desc: "[in] Offset to use when executing kernel."
+    - type: "const size_t*"
+      name: pGlobalWorkSize
+      desc: "[in] Global work size to use when executing kernel."
+    - type: "const size_t*"
+      name: pLocalWorkSize
+      desc: "[in] Local work size to use when executing kernel."
+    - type: uint32_t
+      name: numSyncPointsInWaitList
+      desc: "[in] The number of sync points in the provided dependency list."
+    - type: "const $x_exp_command_buffer_sync_point_t*"
+      name: pSyncPointWaitList
+      desc: "[in][optional] A list of sync points that this command depends on."
+    - type: "$x_exp_command_buffer_sync_point_t*"
+      name: pSyncPoint
+      desc: "[out][optional] sync point associated with this command"
+returns:
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_EXP
+    - $X_RESULT_ERROR_INVALID_KERNEL
+    - $X_RESULT_ERROR_INVALID_WORK_DIMENSION
+    - $X_RESULT_ERROR_INVALID_WORK_GROUP_SIZE
+    - $X_RESULT_ERROR_INVALID_VALUE
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_SYNC_POINT_EXP
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_SYNC_POINT_WAIT_LIST_EXP
```

Comment:
Should this have a trailing colon? `$X_RESULT_ERROR_INVALID_COMMAND_BUFFER_SYNC_POINT_WAIT_LIST_EXP:` just looking at how some of the other errors with sub-bullets are structured

Reply:
yes, good catch!

Reply:
Fixed, thanks!